### DISCUSSION TITLE
How to align items in a table without header row?

### DISCUSSION THREAD

**Discussion by chimit** [outline/outline#7848] [General]
I want to align the rows to the center but keep the header row aligned to the left. Looks like I can only align the whole column which includes all rows. Is it possible to have different alignments for different rows (cells)?

<img width="418" alt="image" src="https://github.com/user-attachments/assets/bce6aebd-c11d-4e77-b691-8227f8af238f">

**Comment by tommoor** [comment]
It's not possible at this time

**Comment by chimit** [comment]
I found a trick to make it work. Looks like copied cells persist alignment information so you can do the following:

1. Align the whole column center
2. Select and copy/cut only the rows of that column
3. Change column alignment to back to default (left)
4. Select the rows of the column and paste

<img width="135" alt="image" src="https://github.com/user-attachments/assets/2548e515-86f5-4690-85c7-ea172baec897">

Of course, this is hard to use and it would be great if the default context menu always displayed the alignment options. Somewhere here:

<img width="469" alt="image" src="https://github.com/user-attachments/assets/98bcaf8a-6758-4d9c-b757-49d16bae3921">

@tommoor, please, consider adding it.
